.github/workflows/verible-ci.yml

```diff
@@ -176,14 +172,17 @@ jobs:
         verbose: true
       continue-on-error: true
 
-    - name: Deployment
-      if: github.ref == 'refs/heads/master' && github.event_name != 'pull_request' && matrix.mode == 'compile'
-      uses: JamesIves/github-pages-deploy-action@3.7.1
+    - name: Compress bazel-bin 
```

Comment:
I'd probably call 'pack' instead of 'compress', as we only do that with the tar.

Reply:
Done.